app/views/events/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="container">
+    <h2>All Events</h2>
+
+    <div class="d-flex justify-content-end mb-3">
+      <button type="button" class="btn btn-primary"><%= link_to 'Create Event', new_event_path, class: 'white' %></button>
+    </div>
+
+
+    <div class="card">
+
+      <div class="card-title">
+        <h3 class="pl-4 pt-4">Upcoming Events</h3>
+      </div>
+
+<div class="card-body">
+      <table class="table table-striped">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Description</th>
+            <th>Creator</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @future.each do |event| %>
```

Comment:
Why didn't we use partial to show event collections? Let's convert our two `each` statement (for `@future` and `@past`) to just render an event partial.

see Section [3.4.5 - 3.4.8](https://guides.rubyonrails.org/layouts_and_rendering.html) of the Rails Guides on how to do this.

This will make your index page more readable and cleaner, eliminating repetitive code